src/hooks/services/hooks.service.ts

```diff
@@ -169,6 +171,16 @@ export class HooksService {
       }
     }
 
+    if (!synchronizationCompleted) {
+      const err = new Error('Synchronization failed');
+      this.logger.error({
```

Comment:
Error logs should be only displayed if an unexpected error occurs, please use a warn log

```suggestion
      this.logger.warn({
```